web/styles/popovers.css

```diff
@@ -788,6 +799,7 @@ ul {
         text-align: center;
         background-color: hsl(0deg 0% 0%);
         border-radius: 0 0 6px 6px;
+        min-height: 25px;
```

Comment:
What's this change for?

Reply:
This prevents the footer from experiencing height fluctuations at the moment when the image is uploaded. 
![2023-06-22 at 17 56 51 - Plum Koi](https://github.com/zulip/zulip/assets/53193850/3439db76-f636-42e1-9bf9-cd7a023f593c)

Reply:
I think this would ideally be better handled by a height attribute on the `<img>` tag, but in this situation, Dan and I discovered that Bootstrap's "height: auto" overrides the attribute. But in any other situation, that would do the job of preserving the vertical height that the image occupies, once loaded.

Reply:
OK I guess this suggests we should consider prioritizing removing that bit of Bootstrap CSS reset, since we now know of a concrete thing it messes up.